.circleci/config.yml

```diff
@@ -22,10 +22,14 @@ jobs:
           command: bazel test //... --config=ci
       - run:
           name: Test REPL for libraries
-          command: bazel-bin/tests/haskell_repl/haskell_lib_repl -e ":quit"
+          command: |
+            bazel build //tests/repl-targets:hs-lib-repl
```

Comment:
Does this have to be a two-line sequence, or as a user, can I `bazel run --direct_run //tests/repl-targets:hs-lib-repl`?

Reply:
I don't have Bazel 0.12 on my machine and CI doesn't have it either, so I think we should use what works with all versions. I don't know what `--direct-run` sets as current working directory and that's important for this script.